### ISSUE TITLE
Improve header of new HTML report: Test INPUT contains confusing informations

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/teamengine#336]
E.g. following output is generated by GeoPackage 1.2 NSG test suite:

```
ics : Core,Tiles,Features,Attributes,Extension Mechanism,Non-Linear Geometry Types,RTree Spatial Indexes,Tiles Encoding WebP,Metadata,Schema,WKT for Coordinate Reference Systems,Tiled Gridded Elevation Data,NSG
iut : file:////srv/local/teamengine-te2/TE_BASE/users/dhintz/s0002/NSG_OGC_Geopackage_VectorWritingTest_SafeSoftware.gpkg 
```

```ics``` term is confusing for users. Also, ```iut``` is not self explanatory and should be replaced by other term.

**Comment by keshav-nangare** [comment]
@dstenger 

If we have multiple test inpts how we can handle it?

```
E.g.

Test INPUT:
getfeatureinfo_in_english : true
vector : false
getmap_exception_in_english : true
capabilities_in_english : true
wms : http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wms130?request=GetCapabilities&service=WMS
getfeatureinfo_exception_in_english : true
```

**Comment by dstenger** [comment]
Can each sub-value be mapped individually? E.g. wms = Tested instance, ...

**Comment by keshav-nangare** [comment]
Yes, we can iterate and map to sub-values from the list.
create list= [iut,wm,wfs-uri,....etc]

map the sub-values i.e. wms with list and if matches then replace with "Tested instance".

**Comment by dstenger** [comment]
Yes, I agree to the proposed solution.

**Comment by keshav-nangare** [comment]
@dstenger 


I have updated the testinput terms in the new HTML report. 
Following is the screenshot after the changes:

![image](https://user-images.githubusercontent.com/14732650/39927664-9f19b64c-5550-11e8-9fd8-b705153b4ad7.png)

**Comment by dstenger** [comment]
@keshav-nangare 

This looks good. Can you please create a pull request?

**Comment by keshav-nangare** [comment]
Updated with this commit: e548163bf7f7c0c511659a0030e32431e04b9a4d [PR: #343]

**Comment by dstenger** [comment]
@keshav-nangare 

Enhancement is not working correctly. If any other title than ```iut,gml,wfs-uri,wmts,wfs,wms``` or ```ics``` is existing, no (empty) title is written to HTML output. WMTS test suite was used for testing.

**Comment by dstenger** [comment]
Screenshot of the current behaviour:

![test-input-wmts](https://user-images.githubusercontent.com/2989166/39991262-397c5b34-576f-11e8-88b4-30e34944fd8e.png)

**Comment by keshav-nangare** [comment]
Fixed with this commit: 3ea7228d72bf0ee0ed1cc5b6f47665c908a5123a

**Comment by dstenger** [comment]
Testing was successful now.
